testgrid/migrations/kustomize/overlays/okteto/migrations.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: migrations
+  labels:
+    app: migrations
+    tags.datadoghq.com/env: "dev"
+spec:
+  selector:
+    matchLabels:
+      app: migrations
+  template:
+    metadata:
+      labels:
+        app: migrations
+        tags.datadoghq.com/env: "dev"
+    spec:
+      volumes:
+        - name: migrations
+          emptyDir:
+            medium: Memory
+      containers:
+      - image: tgmigrations
+        name: pause
+        command:
+          - sleep
+          - infinity
+        volumeMounts:
+          - name: migrations
+            mountPath: /home/schemahero/migrations
+        env:
+          - name: SCHEMAHERO_DRIVER
+            value: postgres
+          - name: SCHEMAHERO_SPEC_FILE
+            value: /home/schemahero/tables
+          - name: SCHEMAHERO_OUT
+            value: /home/schemahero/migrations/plan.yaml
+          - name: SCHEMAHERO_DDL
+            value: /home/schemahero/migrations/plan.yaml
+          - name: SCHEMAHERO_URI
+            value: postgres://testgrid:password@testgrid-postgres:5432/testgrid?sslmode=disable
```

Comment:
does this actually need env vars if its just sleeping forever?

And I presume that this is a deployment so that it gets rerun when things change?